### ISSUE TITLE
Questions about the ideal LOSS for this project.

### ISSUE THREAD

**Issue by jujeongho0** [imoneoi/openchat#105]
Hi.
Thanks for Great Work!!! :)

I'm curious about the train loss when implementing this learning code normally.
If I get a loss of 1.x when not using the weighted_loss, is it correct that I get a loss of 0.1x when using weighted_loss (if I give alpha, the reward value of the GPT3 data, a value of 0.1, as in the paper)?

**Comment by imoneoi** [comment]
Train loss varies a lot across base models and datasets. Ideally, training loss should be less than 1, often 0.4 to 0.8.

In our implementation, we divide the total loss by sum of total weights, so the loss should be at the same level.
